src/form/bf_mx.jl

```diff
@@ -15,59 +15,26 @@ end
 
 ""
 function variable_tp_voltage_prod_hermitian(pm::_PMs.GenericPowerModel{T}; n_cond::Int=3, nw::Int=pm.cnw, bounded = true) where T <: AbstractUBFForm
-    n_diag_el = n_cond
-    n_lower_triangle_el = Int((n_cond^2 - n_cond)/2)
-    for c in 1:n_diag_el
-        PowerModels.variable_voltage_magnitude_sqr(pm, nw=nw, cnd=c, bounded=bounded)
-    end
-
-    wmaxdict = Dict{Int64, Any}()
-    for i in _PMs.ids(pm, nw, :bus)
-        wmax = _PMs.ref(pm, nw, :bus, i, "vmax").values*_PMs.ref(pm, nw, :bus, i, "vmax").values'
-        wmaxltri = _mat2ltrivec!(wmax)
-        wmaxdict[i] = wmaxltri
-    end
+    bus_ids = collect(_PMs.ids(pm, nw, :bus))
 
-    for c in 1:n_lower_triangle_el
-        if bounded
-            _PMs.var(pm, nw, c)[:wr] = JuMP.@variable(pm.model,
-            [i in _PMs.ids(pm, nw, :bus)], base_name="$(nw)_$(c)_wr",
-            lower_bound = -wmaxdict[i][c],
-            upper_bound =  wmaxdict[i][c],
-            start = _PMs.comp_start_value(_PMs.ref(pm, nw, :bus, i), "w_start", c, 1.001)
-            )
-            _PMs.var(pm, nw, c)[:wi] = JuMP.@variable(pm.model,
-            [i in _PMs.ids(pm, nw, :bus)], base_name="$(nw)_$(c)_wi",
-            lower_bound = -wmaxdict[i][c],
-            upper_bound =  wmaxdict[i][c],
-            start = _PMs.comp_start_value(_PMs.ref(pm, nw, :bus, i), "w_start", c, 1.001)
-            )
-        else
-            _PMs.var(pm, nw, c)[:wr] = JuMP.@variable(pm.model,
-            [i in _PMs.ids(pm, nw, :bus)], base_name="$(nw)_$(c)_wr",
-            start = _PMs.comp_start_value(_PMs.ref(pm, nw, :bus, i), "w_start", c, 1.001)
-            )
-            _PMs.var(pm, nw, c)[:wi] = JuMP.@variable(pm.model,
-            [i in _PMs.ids(pm, nw, :bus)], base_name="$(nw)_$(c)_wi",
-            start = _PMs.comp_start_value(_PMs.ref(pm, nw, :bus, i), "w_start", c, 1.001)
-            )
-        end
+    if bounded
+        # get bounds
+        vmax = Dict([(id, _PMs.ref(pm, nw, :bus, id, "vmax").values) for id in bus_ids])
+        vmin = Dict([(id, _PMs.ref(pm, nw, :bus, id, "vmin").values) for id in bus_ids])
+        # create bounded Hermitian matrix variables
+        (Wre,Wim) = variable_mx_hermitian_sqrt_bounds(pm.model, bus_ids, n_cond,
```

Comment:
I think instead of using Wre, Wim, Lre, etc. Wr, Wi, Lr, etc. to be consistent with existing naming conventions, unless there is a compelling reason to do it this way